source/includes/entityref/_shiftposition.md

```diff
@@ -0,0 +1,227 @@
+# ShiftPosition
+
+ShiftPosition is used to create more specific shifts tied to a single job. Also can be used for storing rate information that is tied to the jobShift.
```

Comment:
Change to:
Used to create more specific Shifts tied to a single JobOrder. Also can be used for storing rate information that is tied to the JobShift.

I'm not sure if you want real entity names like used above.